pkg/client/dtclient/config_client.go

```diff
@@ -605,8 +618,9 @@ func unmarshalJson(ctx context.Context, theApi api.API, resp rest.Response) ([]V
 		}
 		for _, kua := range jsonResp.List {
 			values = append(values, Value{
-				Id:   kua.MeIdentifier,
-				Name: kua.Name,
+				Id:           kua.MeIdentifier,
+				Name:         kua.Name,
+				CustomFields: map[string]any{"name": kua.Name, "domain": kua.Domain, "actionType": kua.ActionType},
```

Comment:
i thought about populating CustomFields in general and always, however i think it causes too much memory overhead, compared to what we need it for

Reply:
Agree with this being unnecessary overhead in general - but we could consider if we can use it for any other special case APIs that we already have. e.g. dashboards have some extra fields that are just part of the API value struct now